model/src/main/kotlin/VcsType.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Copyright (C) 2019 Bosch Software Innovations GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonValue
+
+/**
+ * An class for Version Control System types. Each type has one or more [aliases] associated to it, where the first
+ * alias is the definite name.
+ */
+data class VcsType(val aliases: List<String>) {
+    @JsonCreator
+    constructor(type: String): this(
```

Comment:
This constructor could be the primary one., the primary one could be made private and aliases could be made private as well.

Reply:
I had tried that, but it's actually *not* possible because in Kotlin "Data class primary constructor must have only property (val / var) parameters", and I don't want `type` to be a `val`, but only `aliases`.